packages/devtools_app/lib/src/memory/flutter/memory_controller.dart

```diff
@@ -223,9 +261,58 @@ class MemoryController extends DisposableController
 
   final SettingsModel settings = SettingsModel();
 
+  /// Tree to view Libary/Class/Instance
```

Comment:
avoid acronyms. is there a single word that you can use instead of `lci`?
`libraryClassInstanceTreeTable` would be accurate but a bit long :/

Reply:
groupByTreeTable